robotCompletionFinal.cpp

```diff
@@ -0,0 +1,143 @@
+#include "ImageProcessor.hpp"
+
+int core(){
+	double vLeft = 40.0;
+    double vRight = 30.0;
+    char v_go = 20;
+    double Kp = 0.1;
+    char dv = 0;
+    
+    takePicture();
+    SavePPMFile("i0.ppm",cameraView);
```

Comment:
Since the ImageProcessor class takes a picture when it's constructor is called, I think you can remove these line.

Reply:
So remove both 10 and 11?

Reply:
Yes